charts/monitor/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: monitor
+home: https://github.com/deis/monitor
+version: <Will be populated by the ci before publishing the chart>
+description: Monitoring for Deis Workflow.
+maintainers:
+  - name: Deis Team
+    email: engineering@deis.com
+details: |-
+  Workflow is an open source PaaS that makes it easy to deploy and manage applications on your own
```

Comment:
Shouldn't this `details` section be more specific to the monitoring stack, rather than having the same blurb about Workflow everywhere? That would be more consistent with the goal of having the components and their Charts stand alone.

At the very least, I think there should be a meta-issue to ensure that we revisit these and rewrite the details before the first official kubernetes/helm-compatible release of Workflow.